src/WebInterface/Presentation/Http/View/challenge.html.twig

```diff
@@ -0,0 +1,33 @@
+{% extends '@web-interface/layout/condensed.html.twig' %}
+
+{% set page_title = 'Finding your next match...' %}
+
+{% block content %}
+    <h1 class="text-center">
+        Finding your next match<span class="animated-dots"></span>
+    </h1>
+    <div class="container-tight text-center">
+        <p>
+            While we find the best match for you, feel free to explore other games or browse around.
+            Once your match is ready, you'll be automatically redirected.
+        </p>
+        <form action="{{ path('challenge_abort', {gameId: game.id}) }}" method="post" data-abort-form>
+            <button type="submit" class="btn btn-outline-danger">Cancel search</button>
+        </form>
+    </div>
+    <div class="hr-text">Running games</div>
+    <div class="container-tight text-center">
+        <p class="fw-bolder">
+            Feature not implemented yet.
+        </p>
+    </div>
+    <event-source id="event-source"
+                  subscriptions="connect-four-{{ game.id }}{{ game.chatId ? ',chat-' ~ game.chatId }}"
+                  {% if app.environment == 'dev' %}verbose{% endif %}>
+    </event-source>
+    <script type="module">
```

Comment:
Instead of this inline script, a sticky custom element should handle challenges.